docs/source/kit/pi.rst

```diff
@@ -43,8 +43,8 @@ Technical Details
 
 Your robot is running a customised version of the Raspbian_ operating system.
 
-When a USB stick is inserted, the SourceBots software will look for a *main.py*, and then execute it.
+When a USB stick is inserted, the SourceBots software will look for a file named ``main.py``, and then execute it.
 
-The output of your code is written to a ``log.txt`` on the USB stick, and also logged to the *systemd journal*.
```

Comment:
`and also logged to the *systemd journal*.`
This is useful for us, but it might just be confusing for competitors. Should we remove it?